src/styles/docs/styles.css

```diff
@@ -17,6 +17,18 @@
   margin-right: 6px;
 }
 
+/* checkbox */
+.checkboxInlineExample {
+  float: left;
```

Comment:
This should be fine for the docs. Just in general, as much as possible, we should avoid to use `float` to not break the accessibility - tab orders.